distelli-manifest.yml

```diff
@@ -1,4 +1,4 @@
-release-engineering/puppetserver:
+pe-and-platform/puppetserver:
```

Comment:
For future reference, what does this correlate with in Puppet Pipelines? Is it group names?

Reply:
the first part (before the `/`) is a context, similar to AWS contexts

(here is a screenshot of mine, including the "SLV" context, which is kinda like a group)
<img width="415" alt="screen shot 2018-06-18 at 3 43 15 pm" src="https://user-images.githubusercontent.com/7454093/41566041-7bbf20e8-730e-11e8-8857-50d5f15e9ddc.png">

 it does not correspond to a repo, directly.  if that's what you mean...

following through on tony's question...
anyone know whom is in control of `pe-and-platform` and can we get added to it?